### PR TITLE
Add issue and pull request template

### PR DESCRIPTION
**Pull request by divyam3897**
Closes https://github.com/OpenGenus/cosmos/issues/1403

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```
Comment (arnavb) [inline .github/PULL_REQUEST_TEMPLATE.md]: What if this doesn't fix an issue?

**Comment by AdiChat** [comment]
Ideally, our issue for code addition should include information as in [this format](https://github.com/OpenGenus/cosmos/issues/8) 

Moreover, our code addition issues are not solved by any particular pull request.

Kindly make appropriate changes suitable to **Cosmos** 👍

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (divyam3897) [reply .github/PULL_REQUEST_TEMPLATE.md]: so can one submit a PR without even referencing a issue or even if it doesn't has one issue for it?

**Comment by divyam3897** [comment]
@arnavb @AdiChat I have made some changes, feel free to review them and suggest changes if any.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!-- Thanks for filing an issue! Before submitting, please fill in the following information. -->
+
+<!--Required Information-->
+
+**This is a...**
+<!-- choose one by changing [ ] to [x] -->
+- [ ] New algorithm
+- [ ] Update to an existing algorithm
+
```
Comment (arnavb) [inline .github/ISSUE_TEMPLATE.md]: Added a bullet for a genuine issue eg, an error.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Fixes issue #[Mention the issue number it fixes or add the details of the changes if it doesn't has a specific issue.]
+
+Changes: [Add here what changes were made in this issue.]
```
Comment (arnavb) [inline .github/PULL_REQUEST_TEMPLATE.md]: Issue => Pull Request

**Comment by divyam3897** [comment]
@arnavb done 👍

**Comment by AdiChat** [comment]
Looks good 👍  

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**